src/main/resources/types.graphqls

```diff
@@ -22,6 +22,18 @@ input DescendantTargetsFilterInput {
     name: String
 }
 
+input ActiveRecordingFilterInput {
+    name: String
+    state: String
+    continuous: Boolean
+    toDisk: Boolean
+    durationGreaterThanEqual: Long
+    durationLessThanEqual: Long
```

Comment:
Everything here is just in whatever units are used by the backing representation, so yes these are ms. Because of the way that GraphQL works it's not trivial to have it format the durations in different units in responses, and I think it would be the [least surprising](https://en.wikipedia.org/wiki/Principle_of_least_astonishment) for a user if the filter input and the response output use consistent units.

It is probably a good idea to rename these filter keys to include unit information where applicable, though.

Reply:
> It is probably a good idea to rename these filter keys to include unit information where applicable, though.

good idea, makes sense.

Reply:
(this is also a blunder on the `POST /api/v1/targets/:targetId/recordings` API - it interprets the `duration` form attribute as seconds and converts that to milliseconds. It really should have been in milliseconds, or it should have accepted a formatted duration string like `30s`/`500ms`/`1h` etc)